tower-http/src/request_id.rs

```diff
@@ -591,7 +591,7 @@ mod tests {
     #[tokio::test]
     async fn uuid() {
         let svc = ServiceBuilder::new()
-            .set_x_request_id(MakeRequestUuid)
+            .set_x_request_id(MakeRequestUuid::default())
```

Comment:
I'm okay with deriving default but don't think we need to use it here. It's just more typing.

Reply:
Fixed.